backend/src/stwo/prover.rs

```diff
@@ -289,12 +291,13 @@ where
         let component = PowdrComponent::new(
             &mut TraceLocationAllocator::default(),
             PowdrEval::new(self.analyzed.clone()),
+            (SecureField::zero(), None),
```

Comment:
this parameter is for logup, if logup is not used, this is the default value to be passed.

Reply:
comment added. 
Regarding SecureField::zero(), it is not a `const fn`, so I cannot use it directly in a const context. While I could create a function to call it, but this might not be necessary in this case?

Reply:
Oh right, that's not necessary no!